app/src/main/res/layout-sw600dp-land/topic_info_fragment.xml

```diff
@@ -26,7 +26,7 @@
 
         <TextView
           android:id="@+id/topic_name_text_view"
-          android:layout_width="@dimen/elevation_0dp"
+          android:layout_width="@dimen/topic_info_fragment_topic_name_text_view_width"
```

Comment:
This should actually be hardcoded `0dp` directly here. It should not be defined in `strings.xml` file.

Reply:
done and removed that dimension from dimens.xml